lecture_02/assignment_01/eleni_alexi/assignment_01.py

```diff
@@ -0,0 +1,32 @@
+"""Assignment 01: Project box to xy-plane
+"""
+from compas.artists import Artist
+from compas.artists import MeshArtist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+
+# Define a Frame, which is not in the origin and a bit tilted to the world frame
+frame = Frame([10, 4, 8], [1, -0.75, 0], [0, 0.25, -0.2])
+
+# Create a Box with that frame
+box = Box(frame, 4, 3, 5)
+
+# Create a perspective Projection Transformation to project onto the xy-plane from a single point
+P = Projection.from_plane_and_point(Plane.worldXY(), [5, 3, 2])
+
+# Create a Mesh from the Box
+mesh = Mesh.from_shape(box)
+
+# Apply the Projection onto the mesh
+mesh_projected = mesh.transformed(P)
+
+# Create artists
+artist1 = Artist(box)
+artist2 = MeshArtist(mesh_projected)
```

Comment:
While it is valid to create "type-specific artists", like `MeshArtist`, it's usually more convenient to use the generic `Artist` always. Internally compas actually creates an instance of `MeshArtist` anyway.